src/graphql/schemas.js

```diff
@@ -3,102 +3,123 @@ const { gql } = require("apollo-server-express");
 const typeDefs = gql`
   type User {
     id: ID!
-    Email: String!
-    Password: String!
-    FirstName: String!
-    LastName: String!
-    Gender: String
-    Address: String!
-    Latitude: Float!
-    Longitude: Float!
-    Photo: String
-    Events_Owned: [Event]!
-    Events_Invited: [Event]!
-    Events_Attending: [Event]!
```

Comment:
As asked on the readme, are we losing functionality of showing which Events a User has been invited to and/or is Attending here?